app/views/projects/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<H1>Project Details</H1>
+<p>Name: <%= @project.name %></p>
+<p>Material: <%= @project.material %></p>
+<p>Material: <%= @challenge.theme %></p>
```

Comment:
So here you could do `@project.challenge.theme` instead and not need the @challenge instance variable (see comment in controller)